Missing `libgmp3-dev` for Ruby 2.2 on Debian 6

RVM install was fine.  When I tried to install Rails 2.2 it failed with:

E: Unable to locate package libgmp-dev

This was not surprising as libgmp-dev is not a valid package for Debian 6.  Unfortunately we are not in a position to upgrade to 7 at this time.  Is there a workaround or a version of rvm that I can use that is compatible with Debian 6?

Thanks

Same problem here:

`E: Unable to locate package libgmp-dev
++ /scripts/functions/utility : __rvm_try_sudo()  381 > return 100
++ /scripts/functions/requirements/debian : requirements_debian_libs_install()  36 > return 100
Requirements installation failed with status: 100.
make[1]: *** [install] Error 1`

Unfortunately libgmp-dev is called libgmp3-dev in Debian 6 (or at least it
built itself that way when I tried to compile it from source).  I had to
get my sys admin to create a package that named it libgmp-dev and then
everything worked. You could also upgrade your Debian system to Debian 7 or
later, but I did not have that option.  There are some other options
involving patching the rvm package if you're interested -- sorry but I
can't recall the url.

On Thu, May 12, 2016 at 6:39 AM, cespo <notifications@github.com> wrote:

> Same problem here:
>
> E: Unable to locate package libgmp-dev
> ++ /scripts/functions/utility : __rvm_try_sudo() 381 > return 100
> ++ /scripts/functions/requirements/debian :
> requirements_debian_libs_install() 36 > return 100
> Requirements installation failed with status: 100.
> make[1]: *** [install] Error 1
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rvm/rvm/issues/3675#issuecomment-218721422>
>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -27,6 +27,7 @@
 * Installing rbx-3.70 fails on PCLinuxOS 64-bit [\#3895](https://github.com/rvm/rvm/issues/3895)  
 * Can't install Ruby with MacPorts and LibreSSL [\#4208](https://github.com/rvm/rvm/issues/4208)
 * Fix invalid Ruby binary for CentOS 7 ruby-2.4.2 [\#4222](https://github.com/rvm/rvm/issues/4222)
+* Fix invalid `libgmp3-dev` requirement for Debian [\#4238](https://github.com/rvm/rvm/pull/4238)
 
 #### Upgraded Ruby interpreters:
 * Add support for Ruby 2.2.8, 2.3.5 and 2.4.2 [\#4159](https://github.com/rvm/rvm/pull/4159)

--- a/scripts/functions/requirements/debian
+++ b/scripts/functions/requirements/debian
@@ -117,7 +117,7 @@ requirements_debian_define_base()
   requirements_check "$@" \
     make libc6-dev patch openssl ca-certificates \
     curl zlib1g zlib1g-dev libyaml-dev \
-    libsqlite3-dev sqlite3 autoconf libgmp-dev \
+    libsqlite3-dev sqlite3 autoconf \
     libgdbm-dev libncurses5-dev automake libtool bison pkg-config libffi-dev
 
   requirements_${_system_name_lowercase}_define_libgmp

